web/src/main/kotlin/no/nav/su/se/bakover/web/routes/søknad/lukk/LukkSøknadInputHandler.kt

```diff
@@ -0,0 +1,74 @@
+package no.nav.su.se.bakover.web.routes.søknad.lukk
+
+import arrow.core.Either
+import arrow.core.left
+import arrow.core.right
+import no.nav.su.se.bakover.common.deserialize
+import no.nav.su.se.bakover.domain.NavIdentBruker
+import no.nav.su.se.bakover.domain.Søknad
+import no.nav.su.se.bakover.service.søknad.LukkSøknadRequest
+import java.time.LocalDate
+import java.util.UUID
+
+sealed class LukketJson {
```

Comment:
Har det egentlig noen hensikt å bruke en sealed class for inkomne json-typer? Jeg bruker bare la den være 1-1 med typen i frontend og mappe til domene-klassen, som er sealed class.